src/github.com/getlantern/fronted/direct.go

```diff
@@ -117,6 +117,24 @@ func (d *direct) NewDirectHttpClient() *http.Client {
 	}
 }
 
+// Do continually retries a given request until it succeeds because some fronting providers
+// will return a 403 for some domains.
+func (d *direct) Do(req *http.Request) (*http.Response, error) {
+	for i := 0; i < 6; i++ {
+		client := d.NewDirectHttpClient()
```

Comment:
Do we have to re-create the http.Client during each iteration or could we put this outside of the loop?